ui/src/app/shared/services/view-preferences-service.ts

```diff
@@ -21,19 +21,20 @@ export interface PodViewPreferences {
     hideUnschedulable: boolean;
 }
 
+export interface HealthStatusBarPreferences {
+    showHealthStatusBar: boolean;
+}
+
 export type AppsListViewType = 'tiles' | 'list' | 'summary';
 
 export class AppsListPreferences {
     public static countEnabledFilters(pref: AppsListPreferences) {
-        return [pref.clustersFilter, pref.healthFilter, pref.labelsFilter, pref.namespacesFilter, pref.projectsFilter, pref.reposFilter, pref.syncFilter].reduce(
-            (count, filter) => {
-                if (filter && filter.length > 0) {
-                    return count + 1;
-                }
-                return count;
-            },
-            0
-        );
+        return [pref.clustersFilter, pref.healthFilter, pref.labelsFilter, pref.namespacesFilter, pref.projectsFilter].reduce((count, filter) => {
```

Comment:
it looks like `pref.reposFilter` and `pref.syncFilter` got removed here. Was this intentional?

Reply:
yep that was definitely an accidental deletion, nice catch. I've updated that now!